Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,89 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
+
+    	<div class="display">
+        Time:
+            <p id = "Time1"></p> 
+            <p id = "Time2"></p> 
+            <p id = "Time3"></p> 
+            <p id = "Time4"></p>
+            <p id = "Time5"></p> 
+            <p id = "Time6"></p> 
+            <p id = "Time7"></p> 
+            <p id = "Time8"></p>
+        <br><br><br>
+    	</div>
+
+    	<div class="display">
+        Total CPUs: 
+        <p id = "totalCPU"></p>
+        <br> <br>
+    	CPU Usage:
+            <p id = "cpuUsage1"></p> 
+            <p id = "cpuUsage2"></p> 
+            <p id = "cpuUsage3"></p> 
+            <p id = "cpuUsage4"></p>
+    	</div>
+
+    	<div class="display">
+    	Total RAM:
+            <p id = "totalRAM1"></p>
+            <p id = "totalRAM2"></p>
+            <p id = "totalRAM3"></p>
+            <p id = "totalRAM4"></p>
+            <p id = "totalRAM5"></p>
+    		<br><br>
+    	Used RAM:
+            <p id = "usedRAM1"></p>
+            <p id = "usedRAM2"></p>
+            <p id = "usedRAM3"></p>
+            <p id = "usedRAM4"></p>
+            <p id = "usedRAM5"></p>
+    	</div>
+    </div>
+    	<br><br>
+
+        <script>
+        function loadDoc() {
+          var xhttp = new XMLHttpRequest();
+          xhttp.onreadystatechange = function() {
+            if (this.readyState == 4 && this.status == 200) {
+              document.getElementById("Time1").innerHTML = this.responseText[0];
+              document.getElementById("Time2").innerHTML = this.responseText[1];
+              document.getElementById("Time3").innerHTML = this.responseText[2];
+              document.getElementById("Time4").innerHTML = this.responseText[3];
+              document.getElementById("Time5").innerHTML = this.responseText[4];
+              document.getElementById("Time6").innerHTML = this.responseText[5];
```

Comment:
So a couple things here.

The jQuery library can simplify this call! The convenient dollar sign provided by jQuery lets us use CSS selectors to select elements. So to select the element with the `Time1` id, we can use `$("#Time1")` (note the `#` symbol), instead of `document.getElementById("Time1")`.

And the second thing, individual elements aren't required for each character. A single element could be used and then take a substring from the response text. 

So for example, both those points together, something could look like:

```js
var fruits = "applebanana";
$("#firstFruit").text(fruits.substring(0, 5));
```